Stratis.SmartContracts.Tests/UInt256Tests.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using Xunit;
+
+namespace Stratis.SmartContracts.Tests
+{
+    public class UInt256Tests
+    {
+        [Fact]
+        [Trait("UnitTest", "UnitTest")]
+        public void CanConvertToFromUlong()
+        {
+            UInt256 x = 50;
+            Assert.Equal((ulong)50, (ulong)x);
+        }
+
+        [Fact]
+        [Trait("UnitTest", "UnitTest")]
+        public void Uint256FromTooLargeUlongThrowsError()
+        {
+            UInt256 x = UInt256.Parse("010000000000000000");
+            Assert.Throws(typeof(OverflowException), () => (ulong)x);
+        }
+
+        [Fact]
+        [Trait("UnitTest", "UnitTest")]
+        public void CanConvertToFromBytes()
+        {
+            UInt256 x = UInt256.Parse("ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
+            Assert.Equal(x, new UInt256(x.ToBytes()));
+        }
+
+        [Fact]
+        [Trait("UnitTest", "UnitTest")]
+        public void CanAdd()
+        {
+            UInt256 v1 = new UInt256("0000000000000000000000000000000000000000000000000000000000000325");
+            UInt256 v2 = new UInt256("0000000000000000000000000000000000000000000000000000000000000F0F");
+            UInt256 v3 = v1 + v2;
+
+            Assert.Equal(new UInt256("0000000000000000000000000000000000000000000000000000000000001234"), v3);
+        }
+
+        [Fact]
+        [Trait("UnitTest", "UnitTest")]
+        public void CanSubtract()
+        {
+            UInt256 v1 = new UInt256("0000000000000000000000000000000000000000000000000000000000001234");
+            UInt256 v2 = new UInt256("0000000000000000000000000000000000000000000000000000000000000325");
+            UInt256 v3 = v1 - v2;
```

Comment:
Add another test for proving that underflow case throws `System.OverFlowException` for `-` operator.